### ISSUE TITLE
Use `sample_points()` from geopandas

### ISSUE THREAD

**Issue by kadyb** [kadyb/vector-benchmark#10]
Replace our own function with the built in version 0.13

https://github.com/kadyb/vector-benchmark/blob/main/geopandas/sample.py

**Comment by martinfleis** [comment]
Agree, but it should not make a lot of difference as what I wrote initially for this benchmark made its way to geopandas, so the implementation is nearly the same.
